core/kogitoservice/probe.go

```diff
@@ -50,82 +66,61 @@ var defaultProbeValues = corev1.Probe{
 
 // getProbeForKogitoService gets the appropriate liveness (index 0) and readiness (index 1) probes based on the given service definition
```

Comment:
Nitpick: please adjust the comment, the index 0 and 1 looks a bit confusing to me